esp-hal/src/i2c.rs

```diff
@@ -1618,6 +1640,8 @@ pub trait Instance: crate::private::Sealed {
 
         Ok(())
     }
+
+    /// Checks for I2C transmission errors and handles them.
```

Comment:
You might want to expand on what "handling" means here.

Reply:
Resolved